lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+            next if e.empty?
+
+            if connvalues == connheader.keys.size
+              # Reporting-MTA: dns; 10.138.205.200
+              # Arrival-Date: Thu, 23 Jun 2011 02:29:43 -0700
+              v = dscontents[-1]
+
+              if cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*([^ ]+)\z/)
+                # Final-Recipient: RFC822; userunknown@example.jp
+                if v['recipient']
+                  # There are multiple recipient addresses in the message body.
+                  dscontents << Sisimai::Bite.DELIVERYSTATUS
+                  v = dscontents[-1]
+                end
+                v['recipient'] = cv[1]
+                recipients += 1
+
+              elsif cv = e.match(/\A[Xx]-[Aa]ctual-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*(.+)\z/)
+                # X-Actual-Recipient: RFC822; kijitora@example.co.jp
+                v['alias'] = cv[1]
+
+              elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                # Action: failed
+                v['action'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                # Status: 5.1.1
+                # Status:5.2.0
+                # Status: 5.1.0 (permanent failure)
+                v['status'] = cv[1]
+
+              elsif cv = e.match(/\A[Rr]emote-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Remote-MTA: DNS; mx.example.jp
+                v['rhost'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ll]ast-[Aa]ttempt-[Dd]ate:[ ]*(.+)\z/)
+                # Last-Attempt-Date: Fri, 14 Feb 2014 12:30:08 -0500
+                v['date'] = cv[1]
+
+              else
+                if cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                  # Diagnostic-Code: smtp; 550 5.1.1 RCP-P2
+                  #     http://postmaster.facebook.com/response_codes?ip=192.0.2.135#rcp Refused due to recipient preferences
+                  v['spec'] = cv[1].upcase
+                  v['diagnosis'] = cv[2]
+
+                elsif p =~ /\A[Dd]iagnostic-[Cc]ode:[ ]*/ && cv = e.match(/\A[ \t]+(.+)\z/)
+                  # Continued line of the value of Diagnostic-Code header
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + cv[1]
+                  havepassed[-1] = 'Diagnostic-Code: ' + e
+                end
+              end
+
+            else
+              if cv = e.match(/\A[Rr]eporting-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Reporting-MTA: dns; mx.example.jp
+                next if connheader['lhost'].size > 0
+                connheader['lhost'] = cv[1].downcase
+                connvalues += 1
+
+              elsif cv = e.match(/\A[Aa]rrival-[Dd]ate:[ ]*(.+)\z/)
+                # Arrival-Date: Wed, 29 Apr 2009 16:03:18 +0900
+                next if connheader['date'].size > 0
+                connheader['date'] = cv[1]
+                connvalues += 1
+              end
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->